chart/templates/_backingstore.tpl

```diff
@@ -0,0 +1,22 @@
+{{/*
+  is external etcd enabled?
+*/}}
+{{- define "vcluster.externalEtcd.enabled" -}}
+{{- if and (eq (include "vcluster.distro" .) "k8s") (not .Values.controlPlane.backingStore.embeddedEtcd.enabled) -}}
+{{- true -}}
+{{- else if and (eq (include "vcluster.distro" .) "eks") (not .Values.controlPlane.backingStore.embeddedEtcd.enabled) -}}
+{{- true -}}
+{{- else if .Values.controlPlane.backingStore.externalEtcd.enabled -}}
+{{- true -}}
+{{- end -}}
+{{- end -}}
```

Comment:
Not sure if worth, most other projects also don't do it like this also because I believe this might render extra spaces. E.g. ingress-nginx is also not inlining else and ifs in there template files (https://github.com/kubernetes/ingress-nginx/blob/main/charts/ingress-nginx/templates/_helpers.tpl)